### PR TITLE
[image_picker_for_web] Add doc comments to point out that some arguments are'nt supported on the web

### PR DESCRIPTION
**Pull request by devj3ns**
The arguments `maxWidth`, `maxHeight` and `imageQuality` of `pickImage()` and the argument `maxDuration` of `pickVideo()` are not supported in web version of the image_picker package.

I added some doc comments to point out that specifying these argument does not change anything on the pickedFile.

### Pre-launch Checklist

- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by ditman** [comment]
@devj3ns this is great! however because of how the code is imported, these code blocks might not be discoverable.

Do you mind adding a "Limitations of the web version" to the README of the package (similar to [this](https://pub.dev/packages/google_maps_flutter_web#limitations-of-the-web-version), or [this](https://pub.dev/packages/connectivity_for_web#limitations-on-the-web-platform)), so the information is right there in the front page of pub?

Thanks!!

**Comment by devj3ns** [comment]
Hey @ditman, I added the information to the README.md file :)

File: packages/image_picker/image_picker_for_web/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.0.0-nullsafety.1
+
+* Add doc comments to point out that some arguments are'nt supported on the web
```
Comment (ditman) [inline packages/image_picker/image_picker_for_web/CHANGELOG.md:3]:
```suggestion
* Add doc comments to point out that some arguments aren't supported on the web
```

**Review by ditman — APPROVED** [review]
Fixed a little typo, let's go with this one!

**Comment by ditman** [comment]
I broke this PR, fixing :)

**Pull request closed without merging**